models/NewsQuery.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace app\models;
+
+use yii\db\ActiveQuery;
+
+/**
+ * @see News
+ */
+class NewsQuery extends ActiveQuery
+{
+    /**
+     * @return $this
+     */
+    public function published()
+    {
+        return $this->andWhere(['status' => News::STATUS_PUBLISHED]);
+    }
+
+    /**
+     * @return $this
+     */
+    public function sortByDefault()
```

Comment:
`recentFirst()`?

Reply:
No. In this case, I do not need to know how the data is sorted.